common/djangoapps/xblock_django/models.py

```diff
@@ -41,27 +41,10 @@ def is_block_type_disabled(cls, block_type):
 
         return block_type in config.disabled_blocks.split()
 
-    @classmethod
```

Comment:
Note to reviewers: this Django model will be completely deleted in the next story. I just can't remove it until everything is transitioned to the new models.